pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -282,28 +282,33 @@ func getContainerLogs(ctx context.Context,
 
 		logName := logFilename(c.Name, isPrevious)
 
-		// Fetch container logs and continue on error
-		logString, err := getContainerLogString(ctx, client, pod, c.Name, isPrevious, buf, &logTailLines)
+		// Fetch full container logs and continue on error
+		logString, err := getContainerLogString(ctx, client, pod, c.Name, isPrevious, buf, nil)
```

Comment:
I was trying to suggest to avoid this logic (loading the full log every time).....

Reply:
So...

1) Get the last 100 lines;
2) Look for stack trace messages;
3) If found one, fetch the log again with logTailLines + offset.

Reply:
I think that the amount of lines is too small, TBH.

Reply:
or...

1. Get the last 100 lines;
2. Look for stack trace messages;
3. If found one, fetch the full log, check for the first match line + offset.

Reply:
Passing nil as the `logTailLines` param is basically give me the whole log if I am not mistaken. https://pkg.go.dev/k8s.io/kubernetes/pkg/apis/core#PodLogOptions

Reply:
Is there any limit for the number of log lines in openshift? Fetching potentially infinite amount of data isn't cool :) But even if there's I'd vote for still adding our own limit.

Reply:
So, yes... That is why initially I had set a limit. I will changes some things later.